sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+We have four entities, so let's generate a CRUD controller. We'll do that for `Question` first. Find a terminal and run:
+
+```
+symfony console make:admin:crud
+```
+
+As you can see, it recognizes our four entities. I'll hit 1 for `App/Entity/Question`. Let this generate into the default directory and with default namespace.
+
+Sweet! This did exactly one thing. It created a new `QuestionCrudController` file. Let's go open that up.
+
+Before we look too deeply into this, head over to the admin page and refresh to see... absolutely no difference. We *do* have a new `QuestionCrudController`, but these CRUD controllers are kind of useless until you link to them from a dashboard. So, back over on a `DashboardController`, down at the bottom... `yield MenuItem`, but instead of link to dashboard, there's a number of other things that you can link to. We want to `LinkToCrud`. We're gonna pass this a couple things: The label... so `questions`... and an icon, `fa fa-question-circle`. That's a FontAwesome icon. And then, most importantly, the entity's class
+name. So `Question::class`. Behind the scenes, EasyAdmin's going to recognize that there's only one CRUD dashboard for this entity, which is `QuestionCrudController`, and it will know to use this. And yes, in theory, you can have multiple CRUD controllers per entity. That's something we'll talk about later.
+
+Okay, go refresh to reveal our new link, click and... whoa! This is super cool! We have an `isApproved` slider, which saves automatically. We also have a search bar on top... and sortable columns to help us find whatever we're looking for.
+
+We can edit, delete the edit, and it even has a cool calendar. We are absolutely loaded with awesome features. It's pretty cool! So let's repeat this for our other three controllers.
+
+I'll head back to my terminal and run:
+
+```terminal
+Symfony console make:admin:crud
+```
+
+Once again, let's get one for `Answer` with the defaults stuff... And one for `Topic` with the default stuff. I'll clear my screen... And the last one we need is for `User`.
+
+Beautiful! The only thing it did is three more of these CRUD controller classes, but to make those useful, we need to link to them. So, I'm going to paste three more links here. Let's say `Answers`... And I'll customize each of these links.
+
+Nice! I'll refresh... and look! We now have four different admin sections. All right. I want to look just a little deeper into how this is working behind the scenes. Go to `QuestionCrud` controller and look at its base class. I'll hold "cmd + ctrl" and jump into `AbstractCrudController`. We saw earlier that our dashboard extends an `AbstractDashboardController`. Our CRUD controllers extend an `AbstractCrudController`.
```

Comment:
Maybe there should be a comma after "All right"?

The original paragraph was phrased odd and it confused me, so this was my best attempt at translating.

I also just noticed an instance of leaving "controller" out of `QuestionCrudController`. I have fixed that in my draft.

Hopefully the way I wrote "cmd + ctrl" is ok. The "cmd plus" action comes up a few times here, so if I messed it up once, I messed it up everywhere.